slow_odgi/examples.md

```diff
@@ -0,0 +1,157 @@
+# Pollen language walkthrough
+
+## Core types in Pollen
+
+    type Graph = {
+      segments: Set<Segment>;
+      paths: Set<Path>;
+      links: Set<Links>; 
+    }
+
+    type Segment = {
+      sequence: Strand; //ACTGAC, etc. 
+      links: Set<Link>; //links encode their direction + orientation
+      steps: Set<Step>; //steps that go through segment
+    }
+
+    type Handle = {
+      segment: Segment; 
+      orientation: bool;
+    }
+
+    type Path = {
+      steps: List<Step>;
+    }
+
+    type Step = {
+      path: Path; 
+      idx: int; //where in the path is this sequence?
+      handle: Handle; //segment + orientation
+    }
+
+    type Base = A | C | T | G | N
+    type Strand = list Base
+
+    type Link = {
+      start: Handle; 
+      end: Handle;
+    }
+
+## Chop
+
+## Crush
+
+    graph out_g;
+    parset out_segs[Segment, out_g];
```

Comment:
I kinda forget what the two arguments in the square brackets here are supposed to convey. Is the second one always a graph, for example?

Reply:
This is based off of syntax I got from @susan-garry, but I think the first argument is the graph set that is being modified (i.e., `Segment`s or `Path`s and the second is the graph to which that set belongs.